cfgov/retirement_api/jinja2/retirement_api/claiming.html

```diff
@@ -38,6 +38,21 @@ <h1>{{ _("Planning your Social Security claiming age") }}</h1>
                     <a href="es/">Ver página en español</a>
                     {% endif %}
                 </p>
+                <div class="o-well">
+                    {% if current_language == 'en' %}
+                    <p>The Social Security Administration offers
+                         <a href="https://www.ssa.gov/multilanguage/interpreter.html">free interpreter services</a>
+                         and a <a href="https://www.ssa.gov/site/languages/en/">multilanguage gateway</a>
+                         to help consumers who speak other languages.
+                    </p>
+                    {% else %}
+                    <p>La Administración del Seguro Social ofrece
+                         <a href="https://www.ssa.gov/multilanguage/interpreter.html">servicios de intérprete</a>
+                         y un <a href="https://www.ssa.gov/site/languages/en/">portal multilingüe</a>
```

Comment:
Does it make sense to have these link to the Spanish versions of these pages?

```suggestion
                         <a href="https://www.ssa.gov/site/languages/es/interpreter.html">servicios de intérprete</a>
                         y un <a href="https://www.ssa.gov/site/languages/es/">portal multilingüe</a>
```